ui/app/templates/components/oidc/provider-form.hbs

```diff
@@ -93,6 +93,8 @@
         @onChange={{this.handleClientSelection}}
         @disallowNewItems={{true}}
         @fallbackComponent="string-list"
+        @passObject={{true}}
+        @objectKeys={{array "clientId"}}
```

Comment:
debating just changing all of the oidc model params to snake case for consistency - thoughts? Originally I thought it was ember magic that camelized attrs, but apparently it's a method that was added to the application adapter.

Reply:
personally I like camelCase better, but that's less important than being consistent, whichever it is

Reply:
I do as well...it was something that came up before (I think Jordan asked about it) So might be worth all getting on the same page about so we can be consistent moving forward?